src/test/rgw/rgw_multi/tests.py

```diff
@@ -1188,6 +1188,58 @@ def test_bucket_sync_disable_enable():
 
     zonegroup_data_checkpoint(zonegroup_conns)
 
+@attr('resharded_bucket_sync_disable')
+def test_resharded_bucket_sync_disable_enable():
+    zonegroup = realm.master_zonegroup()
+    zonegroup_conns = ZonegroupConns(zonegroup)
+    buckets, zone_bucket = create_bucket_per_zone(zonegroup_conns)
+
+    for bucket_name in buckets:
+        zonegroup_bucket_checkpoint(zonegroup_conns, bucket_name)
```

Comment:
can skip these bucket checkpoints at the beginning, since no objects have been written yet